pkg/annotations/annotations.go

```diff
@@ -1,108 +1,9 @@
 package annotations
 
 const (
-	// Annotations carries the received Kubelet annotations.
-	Annotations = "io.kubernetes.cri-o.Annotations"
-
-	// ContainerID is the container ID annotation.
-	ContainerID = "io.kubernetes.cri-o.ContainerID"
-
-	// ContainerName is the container name annotation.
-	ContainerName = "io.kubernetes.cri-o.ContainerName"
-
-	// ContainerType is the container type (sandbox or container) annotation.
-	ContainerType = "io.kubernetes.cri-o.ContainerType"
-
-	// Created is the container creation time annotation.
-	Created = "io.kubernetes.cri-o.Created"
-
-	// HostName is the container host name annotation.
-	HostName = "io.kubernetes.cri-o.HostName"
-
-	// CgroupParent is the sandbox cgroup parent.
-	CgroupParent = "io.kubernetes.cri-o.CgroupParent"
-
-	// IP is the container ipv4 or ipv6 address.
-	IP = "io.kubernetes.cri-o.IP"
-
-	// NamespaceOptions store the options for namespaces.
-	NamespaceOptions = "io.kubernetes.cri-o.NamespaceOptions"
-
-	// SeccompProfilePath is the node seccomp profile path.
-	SeccompProfilePath = "io.kubernetes.cri-o.SeccompProfilePath"
-
-	// Image is the container image ID annotation.
-	Image = "io.kubernetes.cri-o.Image"
-
-	// ImageName is the container image name annotation.
-	ImageName = "io.kubernetes.cri-o.ImageName"
-
-	// ImageRef is the container image ref annotation.
-	ImageRef = "io.kubernetes.cri-o.ImageRef"
-
-	// KubeName is the kubernetes name annotation.
-	KubeName = "io.kubernetes.cri-o.KubeName"
-
-	// PortMappings holds the port mappings for the sandbox.
-	PortMappings = "io.kubernetes.cri-o.PortMappings"
-
-	// Labels are the kubernetes labels annotation.
-	Labels = "io.kubernetes.cri-o.Labels"
-
-	// LogPath is the container logging path annotation.
-	LogPath = "io.kubernetes.cri-o.LogPath"
-
-	// Metadata is the container metadata annotation.
-	Metadata = "io.kubernetes.cri-o.Metadata"
-
-	// Name is the pod name annotation.
-	Name = "io.kubernetes.cri-o.Name"
-
-	// Namespace is the pod namespace annotation.
-	Namespace = "io.kubernetes.cri-o.Namespace"
-
-	// PrivilegedRuntime is the annotation for the privileged runtime path.
-	PrivilegedRuntime = "io.kubernetes.cri-o.PrivilegedRuntime"
-
-	// ResolvPath is the resolver configuration path annotation.
-	ResolvPath = "io.kubernetes.cri-o.ResolvPath"
-
-	// HostnamePath is the path to /etc/hostname to bind mount annotation.
-	HostnamePath = "io.kubernetes.cri-o.HostnamePath"
-
 	// SandboxID is the sandbox ID annotation.
 	SandboxID = "io.kubernetes.cri-o.SandboxID"
```

Comment:
@containers/podman-maintainers do we want to preserve that name or change it?

Reply:
We're keeping it exclusively for CRI-O compat, AFAIK? Most of these are legacy from when we wanted to make CRI-O run on Libpod IIRC. If CRI-O doesn't use it or uses a different label then we have no reason not to change (or remove entirely?)

Reply:
CRI-O uses the annotation to track the sandbox ID of a particular container. I assume other projects do was well:

https://github.com/containers/podman/blob/be25514b5fb4b65efbf6ec75c304d6b52cf7ed65/pkg/specgen/generate/container.go#L206-L228

I feel that we regress something if we remove/change it.